kubejs/server_scripts/tags.js

```diff
@@ -14,6 +14,8 @@ e.add(`curios:curio`, [`simplemagnets:basicmagnet`, `simplemagnets:advancedmagne
 e.add(`curios:angelring`, [`botania:flight_tiara`]);
 e.add(`forge:ores/certus_quartz`, [`appliedenergistics2:quartz_ore`, `appliedenergistics2:charged_quartz_ore`]);
 e.add(`forge:ores/dimensionalshard`, /rftoolsbase:dimensionalshard_.*/);
+e.add(`forge:gems`, [`quark:biotite`, `psi:psigem`, `mekanism:fluorite_gem`, `iceandfire:amythest_gem`, `forbidden_arcanus:end_crystal_gem`, `astralsorcery:aquamarine`, `appliedenergistics2:fluix_crystal`, `appliedenergistics2:charged_certus_quartz_crystal`, `appliedenergistics2:certus_quartz_crystal`,]);
```

Comment:
Please remove the comma at the end, it's invalid syntax.

Reply:
Whoops, good spotting.